models/users/User.ts

```diff
@@ -4,6 +4,7 @@
 import AccountType from "./AccountType";
```

Comment:
You may want to remove it and also the AccountType.ts in user model to avoid confusing.